app/grandchallenge/evaluation/admin.py

```diff
@@ -58,6 +65,84 @@ def clean(self):
         return cleaned_data
 
 
+class ConfigureAlgorithmPhasesForm(Form):
+    phases = ModelMultipleChoiceField(
```

Comment:
All of the fields on the form should be required.

Reply:
They are required by default.